src/app/components/MediaLoader/Placeholder/index.tsx

```diff
@@ -30,31 +36,54 @@ const MediaPlayerPlaceholder = ({
     guidanceMessage,
   } = mediaInfo ?? {};
 
+  const playButton = (
+    <PlayButton
+      css={styles.playButton}
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onClick={() => {}}
+      title={title}
+      datetime={datetime}
+      duration={duration}
+      durationSpoken={durationSpoken}
+      type={type}
+      guidanceMessage={guidanceMessage}
+      className="focusIndicatorRemove"
+    />
+  );
+
+  const experimentPlayButton = (
+    <MediaIndicator
+      title={title}
+      datetime={datetime}
+      duration={duration}
+      durationSpoken={durationSpoken}
+      type={type}
+      guidanceMessage={guidanceMessage}
+    />
+  );
+
+  const guideComponent = (
+    <Guidance
+      css={styles.guidance}
+      guidanceMessage={guidanceMessage}
+      noJsMessage={noJsMessage}
+    />
+  );
+
+  const experimentSignPost = <SignPost title={title} />;
+
   return (
     // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
     <div
       onClick={onClick}
       css={styles.placeholder}
       data-e2e="media-loader__placeholder"
+      className="placeholder"
     >
-      <Guidance
-        css={styles.guidance}
-        guidanceMessage={guidanceMessage}
-        noJsMessage={noJsMessage}
-      />
-      <PlayButton
-        css={styles.playButton}
-        // eslint-disable-next-line @typescript-eslint/no-empty-function
-        onClick={() => {}}
-        title={title}
-        datetime={datetime}
-        duration={duration}
-        durationSpoken={durationSpoken}
-        type={type}
-        guidanceMessage={guidanceMessage}
-        className="focusIndicatorRemove"
-      />
-
+      {experimentStage === Stages.STAGE_3 ? guideComponent : null}
+      {experimentStage === Stages.STAGE_2 ? experimentPlayButton : playButton}
+      {experimentStage === Stages.STAGE_2 ? experimentSignPost : null}
```

Comment:
Reorder stages numerically
```suggestion
      {experimentStage === Stages.STAGE_2 ? experimentPlayButton : playButton}
      {experimentStage === Stages.STAGE_2 ? experimentSignPost : null}
      {experimentStage === Stages.STAGE_3 ? guideComponent : null}
```

Reply:
Heya, we need to keep the order like this so that components render in the order we expect it to for the screen reader UX.